apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-standard-example.component.html

```diff
@@ -30,7 +30,7 @@
 <p>Icon as an addon</p>
 <fdp-input-group name="employee" placeholder="Email">
     <fdp-input-group-addon>
-        <i class="sap-icon--email"></i>
+        <span class="sap-icon--email" title="Email"></span>
```

Comment:
Why did you change it to a span? Also, you probably need `aria-hidden="true"` for a11y purposes. You can check some blog posts about it https://a11y-101.com/development/icons-and-links

Reply:
It's also in the docs part so you can ignore my comment :) Just wanted to say it's a good to have thing :)

Reply:
I added aria-hidden and reverted `<span>` to `<i>` (chnaged to span previously because it makes icon in italic, but that's should be fixes once latest fundamental-styles gets released).
Thanks!